website/shortlinks/models.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.db import models
+
+from django.utils.translation import gettext_lazy as _
+
+
+class ShortLink(models.Model):
+    slug = models.SlugField(
+        unique=True,
+        max_length=32,
+        verbose_name=_("name for the url"),
+        help_text=_(
+            "This is what you use after the root domain. Be mindful that the url "
+            "this creates could be overridden by a website update."
+        ),
+    )
+
+    url = models.URLField(
+        verbose_name=_("redirection target"),
+        help_text=_("The url the user will be redirected to."),
+    )
+
+    immediate = models.BooleanField(
+        default=False,
+        verbose_name=_("redirect without information page"),
+        help_text=_(
+            "Make sure to only do this when users expect an immediate redirect, if "
+            "they expect a Thalia site a redirect will be confusing."
+        ),
+    )
+
```

Comment:
I think a description would be nice. For example for the whatsapp link it could say to email info@thalia.nu if the whatsapp group is full

Reply:
Already feature creep smh

Reply:
No but really, the idea is to have a redirect. If there should be an explanation it would be better to introduce an environment variable that is just used on an explanation page. Not part of the requirements :p

Reply:
Yes okay good point. I was a bit distracted by the redirection page and thought we might as well add it there. But it's true that's not what this is about.